### PR TITLE
fixbug: WSLog is not worked on device

### PR DESCRIPTION
**Pull request by btxkenshin**
fixbug:WXLog is not worked on device of iOS 10+

### CODE REVIEW

**Comment by btxkenshin** [comment]
I have agreed the CLA and also add the email address used for the commit, check it.
Or is there anything i should do?

**Comment by acton393** [comment]
hi  @btxkenshin ， here shows that you had not signed the CLA , could you please re-check it.

Point: your commit id and email is same as the id you signed the CLA.

File: ios/sdk/WeexSDK/Sources/Utility/WXLog.m [hunk 1]
```diff
@@ -41,6 +41,11 @@
 #define XCODE_COLORS_RESET_BG   XCODE_COLORS_ESCAPE_SEQ "bg;" // Clear any background color
 #define XCODE_COLORS_RESET      XCODE_COLORS_ESCAPE_SEQ ";"  // Clear any foreground or background color
 
+#ifdef DEBUG
+#define WXInnerLog(format, ...) printf("[%s] %s [第%d行] %s\n", __TIME__, __FUNCTION__, __LINE__, [[NSString stringWithFormat:format, ## __VA_ARGS__] UTF8String]);
```
Comment (acton393) [inline ios/sdk/WeexSDK/Sources/Utility/WXLog.m]: It should be better to use English here.

Comment (btxkenshin) [inline ios/sdk/WeexSDK/Sources/Utility/WXLog.m]: english version updated, please check

**Comment by btxkenshin** [comment]
@acton393 I tried the CLA again,and it shows that i had signed the CLA.
And the commit id and email is not the primary email of my github account, then i just added the email to my github account. Is that the problem? And how should i do?

**Comment by cxfeng1** [comment]
@btxkenshin  please try #2068, maybe it can fix your problem.

**Pull request closed without merging**